Data/create.sql

```diff
@@ -0,0 +1,63 @@
+--Create Database
+CREATE DATABASE AntiBlogDB
+
+USE AntiBlogDB
+
+CREATE TABLE [Role]
+(
+    Id   INT IDENTITY NOT NULL PRIMARY KEY,
+    Name VARCHAR(50)  NOT NULL
+)
+
+CREATE TABLE [User]
+(
+    Id               INT IDENTITY NOT NULL PRIMARY KEY,
+    Login            VARCHAR(60)  NOT NULL,
+    PasswordHash     VARCHAR(MAX) NOT NULL,
+    RegistrationDate DATETIME     NOT NULL,
+    RoleId           INT FOREIGN KEY REFERENCES [Role] (Id)
+)
+
+CREATE TABLE [Blog]
+(
+    Id          INT IDENTITY  NOT NULL PRIMARY KEY,
+    Name        NVARCHAR(100) NOT NULL,
+    Description NVARCHAR(MAX),
+    OwnerId     INT FOREIGN KEY REFERENCES [User] (Id)
+)
+
+CREATE TABLE [BlogSubscribers]
+(
+    UserId INT NOT NULL,
+    BlogId INT NOT NULL,
+    PRIMARY KEY (UserId, BlogId),
+    FOREIGN KEY (UserId) REFERENCES [User] (Id),
+    FOREIGN KEY (BlogId) REFERENCES [Blog] (Id),
```

Comment:
'ON DELETE' is missing here. Records should have cascade delete behavior on either Blog or Subscriber entity is deleted.